ecommerce_extensions/tenant/extra_scripts.py

```diff
@@ -0,0 +1,94 @@
+"""
+This function gets called during every request by the
+context processor to return all the custom scripts for a specific path.
+"""
+import logging
+import re
+
+import six
+
+#from eox_theming.configuration import ThemingConfiguration
+
+logger = logging.getLogger(__name__)
+
+common_attributes = {'type': {'options': ['inline', 'external']},
+                     'media_type': {'options': ['module', 'text/javascript'], 'default': 1}}
+
+
+def validate_script_attributes(values):
+    """
+    Validate common attributes for external and inline scripts.
+    """
+    script_attributes = {}
+
+    # Validate common attributes for external and inline scripts
+    for key, attr_values in common_attributes.items():
+
+        value = values.get(key)
+
+        try:
+            if value is not None:
+                attr_values['options'].index(value.lower())
+
+                script_attributes[key] = value.lower()
+            else:
+                default = attr_values['default']
+
+                script_attributes[key] = attr_values['options'][default]
+
+        except Exception:  # pylint: disable=broad-except
+            logger.error("Script could not get loaded. '%s' attribute is missing or is an invalid option.", key)
+
+            return None
+
+    # Validate according to the script type
+    attr = 'src'
+    if script_attributes['type'] == 'inline':
+        attr = 'content'
+
+    script_attributes = check_attribute(values, script_attributes, attr)
+
+    return script_attributes
+
+
+def check_attribute(values, script_attributes, attribute):
+    """
+    Validate the existence of an specific attribute.
+    """
+    try:
+        script_attributes[attribute] = values[attribute]
+    except Exception:  # pylint: disable=broad-except
+        logger.error("Script could not get loaded. '%s' attribute is missing.", attribute)
+
+        return None
+
+    return script_attributes
+
+
+def process_scripts(path, options):
+    """
+    Process and loads all the extra scripts for the template
+    rendered during the request.
+    """
+    scripts = options.get('scripts', {})
+
+    for regex, values in six.iteritems(scripts):
```

Comment:
why do you mix items() with six.iteritems(), this extension is only for python 3 we don't need six